assets/js/modules/thank-with-google/components/common/ColorRadio.js

```diff
@@ -58,7 +58,7 @@ export default function ColorRadio() {
 				image={ <SVG /> }
 				onChange={ onChange }
 				checked={ currentColor === colorThemeID }
-				style={ { outlineColor: colorCode } }
+				checkedBorderColor={ colorCode }
```

Comment:
Looking at this now, we don't actually need separate SVGs for all the color choices, we really just need one SVG for all of them, that accepts a primary color and a secondary color. I'll open an issue to address this.